docs/operator-manual/applicationset/Generators-SCM-Provider.md

```diff
@@ -172,6 +172,37 @@ If you want to access a private repository, you must also provide the credential
 
 Available clone protocols are `ssh` and `https`.
 
+## Azure DevOps
+
+Uses the Azure DevOps API to look up eligible repositories based on a team project within an Azure DevOps organization. 
+The default Azure DevOps URL is `https://dev.azure.com`, but this can be overridden with the environment variable `AZURE_DEVOPS_BASE_URL`.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: myapps
+spec:
+  generators:
+  - scmProvider:
+      azureDevops:
```

Comment:
Nit: Microsoft capitalizes the `O` in `DevOps` for their product.

Reply:
All mentions of `DevOps` in code, YAML and docs should now honor the product's capitalization.